### PR TITLE
French language fixes

### PR DESCRIPTION
**Pull request by Jeto143**
It's not finished but I realized this was kept as a draft for almost a year, so might as well fix these for the time being!

### CODE REVIEW

File: lang/fr.json [hunk 1]
```diff
@@ -1,68 +1,68 @@
...
   "types_of_suffering.hint": "Cliquez sur un article ci-dessus pour en savoir plus.",
   "types_of_suffering.dialog_learn_more": "En savoir plus",
-  "types_of_suffering.dialog_go_back": "Retourner",
-  "types_of_suffering.accidents.description": "Les animaux sont souvent victimes d'accidents et de blessures dans la nature. Ils peuvent être blessés par des prédateurs ou lors de conflits intraspécifiques pour le territoire ou l'accouplement ; brûlés par des incendies ou gelés par des gelées soudaines ; piégés par des terrains difficiles tels que des mares de boue ou des lacs gelés, et confrontés à une mort douloureuse et prolongée ; ou ils peuvent simplement avoir un accident dans le cours normal de leur vie, tout comme les êtres humains. Cependant, contrairement aux humains, les animaux sauvages disposent rarement d'une aide efficace en cas d'accident ou de blessure. Ils se retrouvent presque sans défense face à des animaux plus forts, à des conditions climatiques extrêmes et à des pièges naturels. Néanmoins, les humains parviennent parfois à sauver des animaux blessés ou piégés, même dans des circonstances difficiles.",
+  "types_of_suffering.dialog_go_back": "Revenir",
```
Comment (whoshallsucceed) [inline lang/fr.json]: I don't have the context where it's being used. Would "Retour" not be better suited?

File: lang/fr.json [hunk 2]
```diff
@@ -174,74 +174,74 @@
...
   "resources.books.ufaw.title": "Physiologie et comportement de la souffrance animale",
-  "resources.books.ufaw.description": "Écrit par un expert en aspects de protection appliquée de la physiologie, ce livre est le premier à aborder les aspects physiologiques de la souffrance chez les animaux. Il explore les différentes causes de la souffrance: inconfort physique, soif et faim, les réponses dans le corps qui conduisent à la souffrance et donne un aperçu de la façon dont la souffrance peut être gérée.",
+  "resources.books.ufaw.description": "Écrit par un expert sur les aspects de protection appliquée de la physiologie, ce livre est le premier à aborder les aspects physiologiques de la souffrance chez les animaux. Il explore les différentes causes de la souffrance: inconfort physique, soif et faim, les réponses dans le corps qui conduisent à la souffrance et donne un aperçu de la façon dont la souffrance peut être gérée.",
```
Comment (whoshallsucceed) [inline lang/fr.json]:
```suggestion
  "resources.books.ufaw.description": "Écrit par un expert sur les aspects de protection appliquée de la physiologie, ce livre est le premier à aborder les aspects physiologiques de la souffrance chez les animaux. Il explore les différentes causes de la souffrance : inconfort physique, soif et faim, les réponses dans le corps qui conduisent à la souffrance et donne un aperçu de la façon dont la souffrance peut être gérée.",
```

File: lang/fr.json [hunk 3]
```diff
@@ -174,74 +174,74 @@
...
+  "resources.podcasts.wildness.title": "Who cares about wild animals? avec Maia Laperle",
+  "resources.podcasts.wildness.description": "Maia présente les valeurs et les principes du bien-être des animaux sauvages tout en interviewant des leaders dans le domaine du bien-être des animaux sauvages. Dans chaque épisode, l'animatrice Maia Laperle explore les concepts fondamentaux du bien-être des animaux sauvages et ce que nous savons et ne savons pas sur la vie dans la nature.",
+  "resources.podcasts.the_reality_check.title": "Brian Tomasik sur la souffrance des animaux sauvages",
+  "resources.podcasts.the_reality_check.description": "Notre invité Brian Tomasik ouvre nos yeux sur la souffrance des animaux sauvages. Brian Tomasik écrit sur l'éthique, le bien-être animal et les scénarios de souffrance dans un futur lointain sur son site web principal, \"Essais sur la réduction de la souffrance\".",
   "resources.podcasts.eighty_thousand_hours.title": "Persis Eskander on wild animal welfare and what, if anything, to do about it",
```
Comment (whoshallsucceed) [inline lang/fr.json]: We don't translate this one?

File: lang/fr.json [hunk 4]
```diff
@@ -174,74 +174,74 @@
...
-  "resources.podcasts.eighty_thousand_hours.description": "Éléphants dans les chaînes dans les cirques itinérants;Les porcs enceintes piégés dans des caisses de la taille d'un cercueil dans les fermes industrielles;Deers vivant dans la nature. Nous devrions accueillir le dernier comme une pause agréable avec l'horreur, non?",
-  "resources.podcasts.philosophy.title": "Kyle Johannsen, Éthique animale sauvage: le problème moral et politique des souffrances animales sauvages ",
-  "resources.podcasts.philosophy.description": "De nombreux animaux sauvages sensibles (ou peut-être sensibles) suivent une stratégie de reproduction par laquelle ils ont un grand nombre de descendants, dont la grande majorité souffrent et meurent rapidement ou souffrent et meurent lentement.",
+  "resources.podcasts.eighty_thousand_hours.description": "Éléphants enchaînés dans les cirques ambulants ; des porces gravides piégées dans des caisses de la taille d'un cercueil dans des fermes industrielles ; cerfs vivant à l'état sauvage. Nous devrions accueillir ce dernier comme une agréable pause dans l'horreur, n'est-ce pas ?",
```
Comment (whoshallsucceed) [inline lang/fr.json]:
```suggestion
  "resources.podcasts.eighty_thousand_hours.description": "Éléphants enchaînés dans les cirques ambulants; truies enceintes piégées dans des caisses de la taille d'un cercueil dans des fermes industrielles; cerfs vivant à l'état sauvage. Nous devrions accueillir ce dernier comme une agréable pause dans l'horreur, n'est-ce pas ?",
```

File: lang/fr.json [hunk 5]
```diff
@@ -174,74 +174,74 @@
...
+  "resources.podcasts.philosophy.title": "Kyle Johannsen, Éthique animale sauvage : le problème moral et politique de la souffrance des aniamux sauvages",
```
Comment (whoshallsucceed) [inline lang/fr.json]:
```suggestion
  "resources.podcasts.philosophy.title": "Kyle Johannsen, Éthique animale sauvage : le problème moral et politique de la souffrance des animaux sauvages",
```

File: lang/fr.json [hunk 6]
```diff
@@ -174,74 +174,74 @@
...
+  "resources.websites.world_animal_protection.description": "\"Nous protégeons les animaux du monde. Nous l'avons toujours fait, et nous le ferons toujours.\" World Animal Protections travaille sur le terrain avec des partenaires locaux pour le plus grand effet. Nous sommes actifs dans plus de 50 pays.",
   "resources.websites.animal_advocacy_careers.title": "Animal Advocacy Careers",
-  "resources.websites.animal_advocacy_careers.description": "Animal Advocacy Careers (AAC) est une organisation qui cherche à aborder les goulots d'étranglement de carrière et de talents dans le mouvement de défense des animaux, en particulier le mouvement des animaux d'élevage. Nous fournissons des services de carrière aux personnes à tous les niveaux d'expérience avec le plaidoyer des animaux.",
+  "resources.websites.animal_advocacy_careers.description": "Animal Advocacy Careers (AAC) est une organisation qui cherche à remédier aux goulots d'étranglement en matière de carrière et de talents dans le mouvement de défense des animaux, en particulier celui des animaux d'élevage. Nous fournissons des services de carrière aux personnes de tous niveaux d’expérience en matière de défense des animaux.",
```
Comment (whoshallsucceed) [inline lang/fr.json]:
```suggestion
  "resources.websites.animal_advocacy_careers.description": "Animal Advocacy Careers (AAC) est une organisation qui cherche à remédier aux goulots d'étranglement en matière de carrière et de talents dans le mouvement de défense des animaux, en particulier celui des animaux d'élevage. Nous fournissons des services de carrière aux personnes de tous niveaux d'expérience en matière de défense des animaux.",
```

**Review by whoshallsucceed — CHANGES_REQUESTED** [review]

File: lang/fr.json (hunk 1 above)
Comment (Jeto143) [reply lang/fr.json]: Yes, absolutely! Thanks

File: lang/fr.json (hunk 3 above)
Comment (Jeto143) [reply lang/fr.json]: We should! Done now.

File: lang/fr.json [hunk 7]
```diff
@@ -1,68 +1,68 @@
...
-  "scale_of_suffering.wild_birds": "10-50 oiseaux sauvages",
-  "scale_of_suffering.wild_mammals": "10-100 mammifères sauvages",
-  "scale_of_suffering.reptiles_and_amphibians": "10-10,000 reptiles et amphibiens",
-  "scale_of_suffering.fish": "1,000 - 100,000 poisson",
-  "scale_of_suffering.earthworms": "10,000 - 10⁶ vers de terre",
-  "scale_of_suffering.terrestrial_arthropods": "600,000 - 7*10⁸ arthropodes terrestres",
-  "scale_of_suffering.insects": "600,000 - 7*10⁸ insectes",
-  "scale_of_suffering.in_existence": "…en existence à tout moment.",
+  "scale_of_suffering.farmed_fish": "... sans compter les poissons d'élevage, dont le nombre est estimé à environ 10-13 par personne.",
+  "scale_of_suffering.in_the_wild": "Cependant... dans la nature :<br /><2>Pour chaque humain</2>, il y en entre :",
```
Comment (whoshallsucceed) [inline lang/fr.json]:
```suggestion
  "scale_of_suffering.in_the_wild": "Cependant... dans la nature :<br /><2>Pour chaque humain</2>, il y en entre :",
```

"il y en a entre" / "il y a entre" ?

**Review by whoshallsucceed — CHANGES_REQUESTED** [review]

**Review by whoshallsucceed — APPROVED** [review]

**Comment by Jeto143** [comment]
@AdrienGiboire Thanks again! I might bother you one more time when I create another PR with additional changes as this didn't cover the whole list of keys.

**Pull request merged**